tests/modules/enroll_activity/app/test_enroll_activity_controller.py

```diff
@@ -51,6 +68,17 @@ def test_enroll_activity_controller_missing_code(self):
         assert response.status_code == 400
         assert response.body == 'Field code is missing'
 
+    def test_enroll_activity_controller_enrollment_already_enrolled(self):
+        repo = ActivityRepositoryMock()
+        usecase = EnrollActivityUsecase(repo)
+        controller = EnrollActivityController(usecase)
+
+        request = HttpRequest(body={'user_id': repo.enrollments[7].user.user_id, 'code': repo.enrollments[7].activity.code})
+
+        response = controller(request)
+
+        assert response.status_code == 403
+        assert response.body == 'That action is forbidden for this Enrollment'
 
```

Comment:
Seria melhor se o teste da "activity not found" se relacionasse ao atributo "code", mas vo deixa essa passa.
test_enroll_activity_controller_activity_code_not_found